src/main/java/com/synopsys/integration/detect/configuration/DetectConfigurationFactory.kt

```diff
@@ -314,6 +314,7 @@ open class DetectConfigurationFactory(private val detectConfiguration: PropertyC
         val dryRun = PropertyConfigUtils.getFirstProvidedValueOrDefault(detectConfiguration, DetectProperties.DETECT_BLACKDUCK_SIGNATURE_SCANNER_DRY_RUN, DetectProperties.DETECT_HUB_SIGNATURE_SCANNER_DRY_RUN)
         val uploadSource = detectConfiguration.getValue(DetectProperties.DETECT_BLACKDUCK_SIGNATURE_SCANNER_UPLOAD_SOURCE_MODE)
         val licenseSearch = detectConfiguration.getValue(DetectProperties.DETECT_BLACKDUCK_SIGNATURE_SCANNER_LICENSE_SEARCH);
+        val individualFileMatching = detectConfiguration.getValue(DetectProperties.DETECT_BLACKDUCK_SIGNATURE_SCANNER_INDIVIDUAL_FILE_MATCHING).name
```

Comment:
Don't convert the enum to a String by calling ".name". The configuration has done the heavy lifting of converting it to an enum for you. It should remain an enum in the BlackDuckSignatureScannerOptions and be converted to a String by the user of the BlackDuckSignatureScannerOptions if need be.